reference/release-notes/1.24.2.md

```diff
@@ -0,0 +1,59 @@
+---
+orphan: true
+---
+# 1.24.2
+
+These release notes cover new features and changes in Anbox Cloud 1.24.2.
+
+Anbox Cloud 1.24.2 is a patch release. To understand minor and patch releases, see [Release notes](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/release-notes/release-notes).
+
+Please see [Component versions](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/component-versions/) for a list of updated components.
+
+## Requirements
+
+See [Requirements](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/requirements/) for details on general and deployment specific requirements to run Anbox Cloud.
+
+## New features
+
+* The `anbox-cloud-appliance config show` command lists the available endpoints and their URLs.<!--AC-3002-->
+* Android security updates for January 2025 (see [Android Security Bulletin - January 2025](https://source.android.com/docs/security/bulletin/2025-01-01) for more information).
+* The Android WebView has been updated to 131.0.6778.200.
+
+## Deprecations
+
+There are no new deprecations announced for 1.24.2. For the list of features or components that were deprecated earlier and are planned to be removed in future releases, see [deprecation notices](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/deprecation-notices/).
+
+## CVEs
+
+The Anbox Cloud 1.24.2 release includes fixes from the respective upstreams and components for the following CVEs:
```

Comment:
@keirthana @adglkh  we need to cover the fact that  Google hasn't pubished all details on the bulletin yet (see https://source.android.com/docs/security/bulletin/2025-01-01) so we cannot validate at this point that we have all patches.